<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit_hour">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>161</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_minute">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>171</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_seconds">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>80</y>
      <width>151</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>351</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
